examples/run_og_usa.py

```diff
@@ -62,6 +62,10 @@ def main():
         "mtry_params": d["mtry_params"],
         "mean_income_data": d["mean_income_data"],
         "frac_tax_payroll": d["frac_tax_payroll"],
+        "r_gov_shift": [-0.005],
+        "r_gov_scale": [0.0],
```

Comment:
@jdebacker. Should these values for `r_gov_shift`, `r_gov_scale`, `zeta_K`, and `zeta_D` just be in the `ogusa_default_parameters.json` file so we don't have to update them in the baseline of the run script?